src/main/java/ErrorMsg.java

```diff
@@ -0,0 +1,28 @@
+public class ErrorMsg {
+    /**
+     * All Error Messages thrown by GrapieExceptions are under this class.
+     */
+    public ErrorMsg() {
+    }
+
+    public static String emptyDescriptionError = "OOPS!!! The description cannot be empty.";
+    public static String invalidNumberError = "OOPS!!! Please input a valid number behind!!";
+    public static String wakarimasenError = "OOPS!!! I'm sorry, but I don't know what that means :-(";
+    public static String eventFormatError = "OOPS!!! Event in wrong format. Please use: event your_event "
+            + "/at YYYY-MM-DD TTTT";
+    public static String deadlineFormatError = "OOPS!!! Deadline in wrong format. Please use: deadline "
+            + "your_deadline /by YYYY-MM-DD TTTT";
+    public static String emptyKeywordError = "OOPS!!! Please input a valid word behind find!!!";
+    public static String noSpaceError = "OOPS!!! Please leave a space between command and details!!";
```

Comment:
I like how you abstracted out your error messages. To improve this strategy, consider creating an exception hierarchy, where each exception has its own default error message. Since you've named the errors already, this should be easy to implement.

eg.
<pre>
public class EmptyDescriptionException extends DukeException {
    public EmptyDescriptionException() {
        super("OOPS!! The description cannot be empty.");
    }

    // To get the message, simply call getMessage() defined in the Exception class.
}
</pre>

Reply:
Thank you for this suggestion! Will do it!